frontend/app/services/subscription-service.server.ts

```diff
@@ -23,15 +23,17 @@ type SubscriptionInfo = z.infer<typeof subscriptionInfoSchema>;
 export const getSubscriptionService = moize(createSubscriptionService, { onCacheAdd: () => log.info('Creating new subscription service') });
 
 function createSubscriptionService() {
+  const { CDCP_API_BASE_URI } = getEnv();
+
   async function getSubscription(sin: string) {
     const auditService = getAuditService();
     const instrumentationService = getInstrumentationService();
     auditService.audit('alert-subscription.get', { sin });
-
+    //76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6
     // TODO: "IT-Security won't like SIN being passed as identifier"
-    // TODO: add CDCP_API_BASE_URI
+
     const userId = sin;
```

Comment:
The API doesn't use SIN as the identifier

Reply:
changed to sub

Reply:
Note that the API doesn't use the `userInfoToken.sub` as the identifier either. We have to query the API by the `userInfoToken.sub` to get the `userId` that can then be used as the `users` identifier.

Currently the API doesn't have that operation available so this can be done in a subsequent PR.

Reply:
Resolving as I have opened another ADO task for querying `users` by the `userInfoToken.sub`.